### ISSUE TITLE
Metadata correction for 2024.emnlp-main.337

### ISSUE THREAD

**Issue by a-kostikova** [acl-org/acl-anthology#4249]
### JSON data block

```json
{
  "anthology_id": "2024.emnlp-main.337",
  "abstract": "Solidarity is a crucial concept to understand social relations in societies. In this paper, we explore fine-grained solidarity frames to study solidarity towards women and migrants in German parliamentary debates between 1867 and 2022. Using 2,864 manually annotated text snippets (with a cost exceeding 18k Euro), we evaluate large language models (LLMs) like Llama 3, GPT-3.5, and GPT-4. We find that GPT-4 outperforms other LLMs, approaching human annotation quality. Using GPT-4, we automatically annotate more than 18k further instances (with a cost of around 500 Euro) across 155 years and find that solidarity with migrants outweighs anti-solidarity but that frequencies and solidarity types shift over time. Most importantly, group-based notions of (anti-)solidarity fade in favor of compassionate solidarity, focusing on the vulnerability of migrant groups, and exchange-based anti-solidarity, focusing on the lack of (economic) contribution. Our study highlights the interplay of historical events, socio-economic needs, and political ideologies in shaping migration discourse and social cohesion. We also show that powerful LLMs, if carefully prompted, can be cost-effective alternatives to human annotation for hard social scientific tasks.",
  "authors": [
    {
      "first": "Aida",
      "last": "Kostikova",
      "id": "aida-kostikova",
      "affiliation": ""
    },
    {
      "first": "Benjamin",
      "last": "Paassen",
      "id": "benjamin-paassen",
      "affiliation": ""
    },
    {
      "first": "Dominik",
      "last": "Beese",
      "id": "dominik-beese",
      "affiliation": ""
    },
    {
      "first": "Ole",
      "last": "Pütz",
      "id": "ole-putz",
      "affiliation": ""
    },
    {
      "first": "Gregor",
      "last": "Wiedemann",
      "id": "gregor-wiedemann",
      "affiliation": ""
    },
    {
      "first": "Steffen",
      "last": "Eger",
      "id": "steffen-eger",
      "affiliation": ""
    }
  ]
}
```

**Comment by nschneid** [comment]
Fixes author order.

Also updates the abstract. @mjpost do you want manual review of abstract changes? Hard to do this without a diff.

**Comment by aidakostikova** [comment]
> Fixes author order.
> 
> Also updates the abstract. [@mjpost](https://github.com/mjpost) do you want manual review of abstract changes? Hard to do this without a diff.

Hi!

Could you please provide an update on this? Is this still in review?

Thanks a lot!

**Comment by weissenh** [comment]
Once the second metadata correction sub-issue is processed (abstract change), this issue can be closed too.

**Comment by mjpost** [comment]
Sorry for the delay. @nschneid, it is too much work to compare abstracts line-by-line. Furthermore, abstract data is optional—we don't have an explicit policy that requires correction if it doesn't match. In the interests of volunteer time, therefore, I think we should instead just read them over to make sure it makes sense (which also provides a check that someone doesn't abuse this to insert something untoward), and then approve it. Which I have done.

**Comment by weissenh** [comment]
@mjpost I've out-sourced author name correction and abstract change into separate issues. The first is already processed, the other I have approved earlier today, so technically there is no need to approve it here too. (I am not sure what our script does with the empty affiliations in this issue JSON block).

**Comment by mjpost** [comment]
@weissenh Oh, okay—any particular reason for that?

We used to allow affiliation changes, but too many people were filing them, creating too much work. The fields are just ignored now.

It has occurred to me that we should ask for ORCID iDs for all authors in the correction field as a way of increasing our harvest...

**Comment by weissenh** [comment]
> Oh, okay—any particular reason for that?

@mjpost Well, one part was that author name change was fast to review and with a separate issue I didn't have to deal with the affiliation field. I postponed the review of the abstract ... only now I got to it. As you noticed, it is more work to review.

Collecting ORCIDs is a good idea, just how much work would it be to review them? Do I only check that the name matches or also affiliation/paper info on orcid.org? We already have quite a large backlog of proper author page requests. I fear that metadata correction issues will be harder to review if one also needs to check the ORCIDs provided make sense.

**Comment by mjpost** [comment]
Makes sense. Ignore the collecting ORCID iDs comment. It was tangential. It would require a change to the user interface and a separate discussion. For metadata corrections, we just check that the names match, for now.
